tests/mocha/block_test.js

```diff
@@ -116,29 +116,29 @@ suite('Blocks', function() {
         blocks.B.unplug(true);
         assertUnpluggedHealFailed(blocks);
       });
-      test('A has multiple inputs', function() {
+      test('Parent has multiple inputs', function() {
```

Comment:
If we are going to change to talk about these as Parent, Middle, Child might be worth it to add a comment above explaining which ones are which.